contrib/completion/bash/docker

```diff
@@ -874,6 +875,7 @@ __docker_complete_capabilities_droppable() {
 		MKNOD
 		NET_BIND_SERVICE
 		NET_RAW
+		RESET
```

Comment:
Bash completion has two distinct sets of capabilities: one for capabilities that are enabled by default and thus can be dropped, and one for those that are not enabled by default and therefore can be added.
Any given capablility should not be contained in both lists.

Reply:
Hmm.. isn't this the same as the `ALL` capability (which is also in both lists?)

Reply:
Ah, yes. I didn't realize this capablity is a special one.